lib/pages/common_widgets/goal_widget.dart

```diff
@@ -24,6 +27,24 @@ class GoalWidget extends StatelessWidget {
       });
     }
 
+    doFinishGoal() {
+      final Future<Map<String, dynamic>> responseMessage = finishOrExtend(context, goal.goalId, "true", "0");
```

Comment:
why isn't the extend variable an int?

Reply:
bc the way I set up the lambda was with queryString, I convert it to an int within the lambda